rhel-8.6/manifest.yaml

```diff
@@ -0,0 +1,187 @@
+# Manifest for RHCOS based on RHEL 8.6
+
+rojig:
+  license: MIT
+  name: rhcos
+  summary: OpenShift 4
+
+variables:
+  distro: "rhel"
+  version: "8.6"
+
+# Include manifests common to all RHEL and CentOS Stream versions
+include:
+  - ../common.yaml
+
+# Starting from here, everything should be specific to RHCOS based on RHEL 8.6
+
+ostree-layers:
+  # Temporary logrotate service and timer units
+  - overlay/15rhcos-logrotate
+  - overlay/25rhcos-azure-udev
+
+# See README.md
+# and https://github.com/openshift/release/blob/master/core-services/release-controller/README.md#rpm-mirrors
+repos:
+  - rhel-8-baseos
+  - rhel-8-appstream
+  - rhel-8-fast-datapath
+  - rhel-8-server-ose
+
+# https://bugzilla.redhat.com/show_bug.cgi?id=1938928
+rpmdb: bdb
+
+# We include hours/minutes to avoid version number reuse
+automatic-version-prefix: "412.86.<date:%Y%m%d%H%M>"
```

Comment:
This is bumping the version from 411 to 412. For commits like this which move a lot of pieces around, it'd be better to keep the end state functionally the same and split out this kind of changes in separate commits with their own rationales.

Reply:
And indeed, agree that we should not sneak changes in. With the many rebases and tests I did I forgot to split those